grisera_api/experiment/experiment_model.py

```diff
@@ -10,10 +13,15 @@ class ExperimentIn(BaseModel):
 
     Attributes:
         name (str): Name of experiment
+        authors (Optional[Set[AuthorIn]]): Authors of the experiment
+        publication(Optional[PublicationIn]): Publication, in which the experiment is described
         abstract (Optional[str]): Summary of the experiment
         additional_properties (Optional[List[PropertyIn]]): Additional properties for experiment
     """
     name: str
+    authors: Optional[List[AuthorIn]]
+    publication: Optional[PublicationIn]
+    # scenario = Optional[List[Activity]]
```

Comment:
Why is it commented? Are we gonna use this scenario later?

Reply:
I wasn't sure what should be included as properties in this task. I have just noticed that I didn't also include frequency property. :(